### DISCUSSION TITLE
How to remove the welcome information in the first opening

### DISCUSSION THREAD

**Discussion by ghost** [nushell/nushell#8297] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/32454224/222757437-e909f6e3-052c-4046-9e1f-3b771b941f68.png)

**Comment by fdncred** [comment, marked as answer]
Set show_banner to false in the config.nu file. There are instructions in our documentation and also the latest version shows that link.

**Reply by ghost** [reply]
Sorry, I just used it. I don't know it yet.Thank you for reply
